src/data/use-cases/db-authentication.ts

```diff
@@ -1,25 +1,31 @@
-import { type LoadGuardianByEmailRepository, type HashComparer, type TokenGenerator, type UpdateAccessTokenRepository, type HashGenerator } from '@/data/protocols'
-import { type Authentication } from '@/domain/use-cases/authentication'
+import { type Authentication } from '@/domain/use-cases'
```

Comment:
Thats good, linter is not warning to long lines. We need cauntion with this.

Reply:
Yes, it is a good pratice a standart line length limit. Usualy linters has rules to this.